src/SWP/Bundle/WebRendererBundle/Theme/Configuration/TenantableConfigurationProvider.php

```diff
@@ -63,14 +63,19 @@ public function __construct(
      */
     public function getConfigurations()
     {
-        $configs = array_map(
-            [$this->loader, 'load'],
-            $this->fileLocator->locateFilesNamed($this->configurationFilename)
-        );
+        // Handle there being no theme.json file
+        try {
```

Comment:
An invalid argument exception is thrown hair during the kernel boot, with the message that "The file "theme.json" does not exist (searched in the following directories: /home/sourcefabric/Public/web-publisher/app/themes)." This is really where we should throw the NoThemeException, but it can't be handled gracefully at this point. A kernel exception listener will not be called until the kernel is booted.